gb-bus/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 
 [dependencies]
 rand = { version = "0.8.4", features = ["small_rng"] }
-serde = { version = "1.0", optional = true }
+serde = { version = "1.0", features = ["std", "derive"], optional = true }
```

Comment:
You're not required to specify the features since they're already defined in the features section for `serialize`